modules/admin_manual/pages/enterprise/file_management/files_lifecycle.adoc

```diff
@@ -0,0 +1,126 @@
+= File Lifecycle Management
+:toc: right
+
+Manage the lifecycle of files within ownCloud to keep your working structure clean, organized and up to date. Choose an archiving and expiration policy that suits your workflow and select an appropriate archive destination to store your old files. Policies can be configured to include and exclude certain files, users
+and groups from the archiving actions. 
+
+== Installation & Configuration
+
+=== Upload Time Tracking
+
+The first part of file lifecycle management begins with knowing when files first enter ownCloud. To start tracking this the app must be enabled. This will store an `upload-time` with each file, which is the server time at which it first appeared on the ownCloud server. Only files are tracked, not folders since these are never archived, only the files within them.
+
+If installing this app on a system with existing data, the `lifecycle:set-upload-time` command can be used to pre-set a specific upload time to all existing files, so they may also be included in the archiving workflow. Files without an `upload-time` will not be considered for archive.
+
+Example: `occ lifecycle:set-upload-time 2019-11-01`
+
+=== Policy Configuration
+
+The Lifecycle app uses policies to determine which files to archive and when, as 
+well as when to expire the files from archive, if at all. Hard and Soft policies 
+are available, depending on the use case requirements.
+
+By default, the SoftPolicy is used. To configure the policy, use the `occ` command:
+
+`occ config:app:set files_lifecycle policy --value='hard'`
+
+Three options are available for controlling the policies, all set via `occ config:app:set` under the `files_lifecycle` app:
+
+- `archive_period` int - The number of days after upload, or restore, that the file will be archived
+- `expire_period` int - The number of days after archive which the file will be  permanently deleted
+- `excluded_groups` string, comma separated group ids - Groups who are exempt from lifecycle policy.
+
+Example: `occ config:app:set files_lifecycle archive_period --value='90'`
+
+==== Soft Policy
+  
+The Soft policy is for use cases where the main working directory  should be reserved for actively used files, and users are encouraged to archive older data. Admins are also still able to impersonate users and restore files for them to assist where necessary.
```

Comment:
`working directory  should` --> `working directory should`
(double blank used between directory and should)